tests/test_project.py

```diff
@@ -373,3 +376,40 @@ def test_archflags_envvar_parsing_invalid(package_purelib_and_platlib, monkeypat
     finally:
         # revert environment variable setting done by the in-process build
         os.environ.pop('_PYTHON_HOST_PLATFORM', None)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 13), reason="Requires Python 3.13 or higher")
```

Comment:
Nit: I think we are standardizing to test skip reasons starting with a lowercase character (they look a bit nicer in pytest output).